client/src/components/ProtectedRoute/ProtectedRoute.tsx

```diff
@@ -0,0 +1,14 @@
+import { useAuth } from '../../context/useAuthContext';
+import { Redirect, Route, RouteProps } from 'react-router-dom';
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface ProtectedRouteProps extends RouteProps {}
```

Comment:
```suggestion
interface ProtectedRouteProps extends RouteProps {
authenticationPath: string;
exact: boolean
}
```

Reply:
Oh thanks, I will definitely add the types here.